odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -41,7 +41,7 @@
     'product.attribute.line': 'product.template.attribute.line',
     'product.attribute.price': 'product.template.attribute.value',
     'product.uom': 'uom.uom',
-    'product.uom.category': 'uom.category',
+    'product.uom.categ': 'uom.category',
```

Comment:
Isn't this already changed in other PR?

Reply:
It was changed in base pre-migration, but not here.